book/05-distributed-git/sections/distributed-workflows.asc

```diff
@@ -39,15 +39,16 @@ With Git's branching model, it's possible for hundreds of developers to successf
 (((workflows, integration manager)))
 Because Git allows you to have multiple remote repositories, it's possible to have a workflow where each developer has write access to their own public repository and read access to everyone else's.
 This scenario often includes a canonical repository that represents the ``official'' project.
-To contribute to that project, you create your own public clone of the project and push your changes to it.
+Each developer has also own private repository for conducting changes in workspace then committing.
+To contribute to that project, you create your own public clone of the project and push your changes to it from your private repository.
 Then, you can send a request to the maintainer of the main project to pull in your changes.
 The maintainer can then add your repository as a remote, test your changes locally, merge them into their branch, and push back to their repository.
 The process works as follows (see <<wfdiag_b>>):
 
 1.  The project maintainer pushes to their public repository.
-2.  A contributor clones that repository and makes changes.
-3.  The contributor pushes to their own public copy.
-4.  The contributor sends the maintainer an email asking them to pull changes.
+2.  A contributor clones that repository and makes then commits changes.
```

Comment:
```suggestion
2.  A contributor clones that repository and commits changes.
```

Reply:
Thank you for improvement. Actually I missed "makes **changes** then commits..."

Reply:
Also this comment from me: text before PR mentions - in explicit way - making changes in point 2 so let retain it there.

Reply:
By that logic, we should be including paragraphs with every numbered point here. This isn't meant to be an exhaustive list of commands the user needs to type in, and readers at this point should know that they can't make commits without making changes. Remember, this is chapter 5, and the reader should be pretty comfortable working with a local repository at this point.